starter-code/styles.css

```diff
@@ -0,0 +1,142 @@
+body {
+  font-family: tahoma;
+}
+
+.containerFluid {
+margin-right: 0;
+margin-left: 0;
+display: flex;
+flex-wrap: wrap;
+justify-content: space-between
+
+}
+.list-inline-item {
```

Comment:
Instead this ".list-inline-item" you can say 
```
nav a {
  // all the same as before
}
```
and you can delete class = "list-inline-item" from your HTML.